service/collector/client.go

```diff
@@ -0,0 +1,47 @@
+package collector
+
+import (
+	"github.com/giantswarm/azure-operator/client"
+	"github.com/giantswarm/azure-operator/service/credential"
+	"github.com/giantswarm/microerror"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	// kubernetesCredentialNamespace is the namespace in which we store credentials.
+	kubernetesCredentialNamespace = "giantswarm"
+
+	// kubernetesLabelSelector is the label selector we use to retrieve credentials.
+	kubernetesLabelSelector = "app=credentiald"
```

Comment:
Would be nice to align those names:
`kubernetesCredentialNamespace`
`kubernetesCredentialLabelSelector`

or even drop the `kubernetes` prefix:
`credentialNamespace`
`credentialLabelSelector`

Reply:
I second that preferring the version w/o prefix. `kubernetesLabelSelector` don't tell much about the content.